src/sync/installation.ts

```diff
@@ -324,12 +326,53 @@ const doProcessInstallation = async (data: BackfillMessagePayload, sentry: Hub,
 		);
 
 		statsd.increment(metricTaskStatus.complete, [`type:${nextTask.task}`, `gitHubProduct:${gitHubProduct}`]);
+		recordTaskProcessDuration(nextTask, gitHubProduct, subscription, data);
 
 	} catch (err) {
 		await handleBackfillError(err, data, nextTask, subscription, logger, scheduleNextTask);
 	}
 };
 
+const recordTaskProcessDuration = (nextTask: Task, gitHubProduct: string, subscription: Subscription, data: BackfillMessagePayload) => {
+	statsd.increment(metricTaskDuration.duration, Math.floor(new Date().getTime() / 1000) - nextTask.taskStartTimeInSec, 1, {
+		task: nextTask.task,
+		gitHubProduct,
+		repoNums: getRepoNumberBuckets(subscription.totalNumberOfRepos),
```

Comment:
So for both `repoNums` and `commitsFromDate` I use manual buckets here. The idea is in the dashboard we can easily filter the tag but `repoNum = 0-9` or `repoNum = 1000+`.
I don't think we should put a raw number here, because the field is `tag` and `tag` isn't supposed to be dynamic and grow unlimited. But correct me if I am wrong. 

ps: Happy to discuss the bucket items as well.